SparkPost/2web2ui
# SA-1005: Add to Watchlist page

### What Changed
 - Created a page at `/blacklist/watchlist/add`
 - Page handles adding a single resource (sending domain or ip address) to the watchlist
 - Page offers ability to "save" and redirect back to `blacklist/watchlist` or to "save and add another" which does the same minus the redirect

### How To Test
 - Add something to the watchlist using save
   - Ensure it redirects 
   - Ensure it displays a success message
 - Add something to the watchlist using save and add another
   - Ensure it does not redirect
   - Ensure it clears input
   - Ensure it displays a success message
 - Try to re-add an existing resource
  - Ensure it displays the error message below the resource
  - Ensure the error message goes away when you change the input
